src/RpcClient/RpcClient.csproj

```diff
@@ -14,7 +14,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Neo" Version="3.0.0-CI00972" />
+    <PackageReference Include="Neo" Version="3.0.0-preview3-00" />
```

Comment:
I think we should use the latest version for nugets in master

Reply:
or do you want to merge for create the tag?

Reply:
> or do you want to merge for create the tag?

Yes, should use the latest version of nuget. I've already created the tag on preview3 branch, no need on master since there will be extra commit on master.

Reply:
Shall we merge first and I upgrade the latest neo dependency in master later? I think they are two workflows after release.

Reply:
I think that we can update them here

Reply:
> I think that we can update them here

I can't do this since `preview3` is the release branch and should not be changed from nuget `3.0.0-preview3-00`. I have to wait until master is merged.

Reply:
ok! approved :)